jetty-websocket/javax-websocket-server/src/test/java/org/eclipse/jetty/websocket/javax/server/WebSocketServerExamplesTest.java

```diff
@@ -95,13 +95,14 @@ public void onError(Throwable cause)
     }
 
     static Server _server;
+    static ServerConnector connector;
     static ServletContextHandler _context;
 
     @BeforeAll
     public static void setup() throws Exception
     {
         _server = new Server();
-        ServerConnector connector = new ServerConnector(_server);
+        connector = new ServerConnector(_server);
         connector.setPort(8080);
```

Comment:
I couldn't figure out how to change this one because it uses xml HttpClient provider
https://github.com/eclipse/jetty.project/blob/0d9de2822670cee7e2ca16ced238594d2c1ea45f/jetty-websocket/javax-websocket-server/src/test/resources/jetty-websocket-httpclient.xml#L3-L18

Reply:
Set the arg with a property: `<Arg><New ...><Arg><Property ..>...` and then set externally